pkgs/by-name/go/go-pray/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  pkgs,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "go-pray";
+  version = "0.1.13";
+
+  src = fetchFromGitHub {
+    owner = "0xzer0x";
+    repo = "go-pray";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-wi/sUjzNbVLbqwtM1BglmXgkaqMamjIsGS6EjlyeC9Y=";
+  };
+  vendorHash = "sha256-qMTg2Vsk0nte1O8sbNWN5CCCpgpWLvcb2RuGMoEngYE=";
+
+  nativeBuildInputs = with pkgs; [
+    pkg-config
+    alsa-lib.dev
+  ];
```

```suggestion
  nativeBuildInputs = [
    pkg-config
    alsa-lib
  ];
```
No need to specify .dev outputs, mkDerivation already pulls them